### PR TITLE
add trigger actions

### PR DESCRIPTION
**Pull request by shved**
I've added a trigger actions to retrieve list of active triggers and trigger a contact.

### CODE REVIEW

**Comment by shved** [comment]
Hi, @lcpriest! Review and merge it please 🔍

File: test/vcr_helper.rb [hunk 1]
```diff
@@ -1,4 +1,3 @@
-require 'pry'
```
Comment (lcpriest) [inline test/vcr_helper.rb:1]: Good catch.

File: lib/autopilot/resources/contact.rb [hunk 2]
```diff
@@ -7,5 +7,6 @@ class Contact < Resource
     include Operations::All
     include Operations::Retrieve
     include Operations::Identify
+    include Operations::TriggerJourney
```
Comment (lcpriest) [inline lib/autopilot/resources/contact.rb:10]: Non RESTful APIs make me sad.

Comment (shved) [reply lib/autopilot/resources/contact.rb:10]: @lcpriest what you mean for non restful here? There is no journey resource and the only thing you can trigger is a jorney. But I feel something is wrong with this modules anyway.

**Pull request merged**